### PR TITLE
Card part attributed text patch

### PR DESCRIPTION
**Pull request by jl-codes**
## Issue Link :link:
 <ul>
   <li> Is this a bug fix or a feature? </li>
   <li> Does it break any existing functionality?</li>
</ul>

## Goals of this PR :tada:
 <ul>
   <li> Why is the change important? </li>
   <li> What does this fix? </li>
   <li> How far has it been tested? </li>
 </ul>
 
## How Has This Been Tested :mag:

Please let us know if you have tested your PR and if we need to reproduce the issues. Also, please let us know if we need any relevant information for running the tests.

<ul>
 <li> User Interface Testing </li>
 <li> Application Testing </li>
</ul>

## Test Configuration :space_invader:

<ul>
 <li> Xcode version: </li>
 <li> Device/Simulator </li>
 <li> iOS version || MacOSX version</li>
</ul>

## Things to check on :dart:

 - [ ] My Pull Request code follows the coding standards and styles of the project 
 - [ ] I have worked on unit tests and reviewed my code to the best of my ability 
 - [ ] I have used comments to make other coders understand my code better 
 - [ ] My changes are good to go without any warnings 
 - [ ] I have added unit tests both for the happy and sad path 
 - [ ] All of my unit tests pass successfully before pushing the PR 
 - [ ] I have made sure all dependent downstream changes impacted by my PR are working

### CODE REVIEW

File: CardParts/src/Classes/Card Parts/CardPartAttributedTextView.swift [hunk 1]
```diff
@@ -206,30 +206,20 @@ public class CardPartAttributedTextView: UIView, CardPartView {
     func setDefaultsForType(_ type: CardPartAttributedTextType) {
         switch type {
         case .small:
-            DispatchQueue.main.async { [weak self] in
-                self?.font = CardParts.theme.smallAttributedTextFont
-                self?.textColor = CardParts.theme.smallAttributedTextColor
-            }
+            font = CardParts.theme.smallAttributedTextFont
+            textColor = CardParts.theme.smallAttributedTextColor
```
Comment (badrinathvm) [inline CardParts/src/Classes/Card Parts/CardPartAttributedTextView.swift]: @jl-codes why is it removed ?

Comment (jl-codes) [reply CardParts/src/Classes/Card Parts/CardPartAttributedTextView.swift]: In the current form, we are unable to override the font and textColor attributes. Removing the DispatchQueue from this allows for us to override these settings in the view controller.

Comment (jl-codes) [reply CardParts/src/Classes/Card Parts/CardPartAttributedTextView.swift]: In the current form, we are unable to override the font and textColor attributes. Removing the DispatchQueue from this allows for us to override these settings in the view controller.

Comment (bcarreon) [reply CardParts/src/Classes/Card Parts/CardPartAttributedTextView.swift]: In didSet of font and didSet of textColor we are already Dispatching on main thread. The problem is that it puts it on the main thread twice

**Pull request merged**